Proposed README.md file using standardized template

riverma opened the pull request:
## Purpose
- New README.md file format
- Leveraging https://nasa-ammos.github.io/slim/documentation/starter-kits/#software-project-template
- NOTE: this is a DRAFT pull-request and it needs to be customized / adapted to your repo - please commit to this PR branch with your customizations before merging!
- NOTE: make your modifications by searching for the string `INSERT` in the file and replacing with appropriate values for your repo
- NOTE: read through the file thoroughly to ensure its well adapted for your repo and your scenario, feel free to modify as needed
## Proposed Changes
- [ADD/CHANGE] README.md file with sample content
## Testing
* For live example, see: https://github.com/riverma/terraformly/blob/main/README.md
* For actual rendering of this current README see fork: https://github.com/riverma/unity-architecture/tree/riverma-readme

riverma committed:
Proposed README.md file / template

riverma committed:
Update README.md

mike-gangl reviewed (commented):
i like the consistent readme pages, but the quickstart, requiement, setup, usage, etc seem... out of place for an architecture document. I wonder if there should be a more general 'unity' landing page, or like you mentioned previously, if this is the one we'll use as the standard. That would make sense with the bug reports, etc.

riverma commented:
> i like the consistent readme pages, but the quickstart, requiement, setup, usage, etc seem... out of place for an architecture document.

@mike-gangl - I can trim the README to remove unnecessary sections (and just FYI you have commit access to the PR branch as well by default, just click the [riverma-readme branch](https://github.com/riverma/unity-architecture/tree/riverma-readme) and modify the `README.md` in the GitHub editor, and commit - the changes will show up here. However, isn't there a build process for generating the HTML versions of your C4 diagrams? Say if someone wanted to contribute a change to the proposed architecture, how would they locally test to ensure it works with your setup? I think that's what the _Quick Start_ is all about in my mind. What do you think?

Also, if the README is _missing_ any key sections you think, let me know. 

> I wonder if there should be a more general 'unity' landing page, or like you mentioned previously, if this is the one we'll use as the standard. That would make sense with the bug reports, etc.

A centralized webpage / landing-page is a great idea. Let me mark that as a to-do for Q3? We can update all the repo READMEs to point to that once its up?

riverma committed:
Update README.md

riverma commented:
@mike-gangl - I've updated the PR for this to be ready-to-merge upon your review. I just set a couple things to TBD that can be filled out later, and cleaned up the unnecessary parts of this readme. 

Please see demo at: https://github.com/riverma/unity-architecture/tree/riverma-readme
